### DISCUSSION TITLE
No autocomplete?

### DISCUSSION THREAD

**Discussion by patlux** [LazyVim/LazyVim#4556] [Q&A]
hey all,

does anyone have the same problem. Sometimes the suggestions are completely off:

![CleanShot 2024-10-08 at 22 22 30@2x](https://github.com/user-attachments/assets/4d1caf6f-4d1e-4bbc-95d5-2c1a55855b3f)

![CleanShot 2024-10-08 at 22 22 17@2x](https://github.com/user-attachments/assets/47a11b29-4472-445b-9185-0c0c0e3630f3)

It doesn't show any prop of `DetailViewItem`, but when I enter the first letter it displays all of them beginning with the letter:

![CleanShot 2024-10-08 at 22 25 36@2x](https://github.com/user-attachments/assets/772e86f8-c172-499b-8b62-72e1fbd6fe6f)

How can I fix this, that all fields are shown initial?

**Comment by dpetka2001** [comment]
If i remember correctly nvim-cmp has an option to trigger the suggestions based on length of characters the user types. Lazyvim does not make any change regarding this. So, maybe it's the default nvim-cmp value that causes this. Read its docs or ask at nvim-cmp repo for how to change that.

**Reply by patlux** [reply]
The problem doesn't happen always, only sometimes.

E.g. now it works after reopening the project after a while:

![CleanShot 2024-10-08 at 23 01 52@2x](https://github.com/user-attachments/assets/f80aac25-5f06-4bd1-bad6-2c256bb7bd48)

I mapped <C-n> to display the autocomplete:

```lua
local cmp = require("cmp")

return {
  "hrsh7th/nvim-cmp",
  opts = {
    mapping = cmp.mapping.preset.insert({
      ["<C-n>"] = cmp.mapping.complete(),
    }),
  },
}
```

**Reply by dpetka2001** [reply]
Then maybe it has to do with how long it takes the server to send the suggestions to `nvim-cmp` (which might depend on how big the project is)? Maybe it starts eating up RAM and that makes the server laggy and less responsive compared to when you first open your project. I'm just guessing.

**Comment by nowaylifer** [comment]
Try `vim.opt.complete = ""`
